modules/client/test/integration/client/methods.test.ts

```diff
@@ -1691,5 +1695,284 @@ describe("Client", () => {
       //   "Invalid ENS name"
       // );
     });
+
+    describe("isDaoUpdateProposalValid", () => {
+      it("should check if a proposal is valid for updating the dao and fail with an invalid proposal id", async () => {
```

Comment:
```suggestion
      it("throws an `InvalidProposalIdError` if the proposal ID is ???", async () => {
```
Please improve the test description.

My recommendation: Do not repeat the part of the description that is already part of the `describe` statement for every test. "isDaoUpdateProposalValid" already conveys thaat this is about checking if a proposal is valid.